README.md

```diff
@@ -325,6 +336,10 @@ beforeStartHub$: Observable<{
 // start hub automatically
 startHub$: Observable<SignalRStartHubAction>;
 ```
+```ts
+// stop hub 
+startHub$: Observable<SignalRStopHubAction>;
```

Comment:
`stopHub$`

Reply:
> Also, you do not seem to have compiled the *.ts file with `tsc`. But that make sense. I need to see how to put that in a CI pipeline like GitHub Actions.

I can do so but I don't think it is best practice to commit compiled output as we can always add "files" key to package.json to include compiled output at release time as most npm modules do. but it is up to you

Reply:
Fixed, thanks

Reply:
I did not think about that when I started the library but clearly it is better to not include the compiled files in the repo. I will fix that.

Reply:
Well, I remove the compiled folder and updated the `.gitignore` file. It's better now. Thanks for the tip.